mmpose/datasets/datasets/face/face_300w_dataset.py

```diff
@@ -0,0 +1,171 @@
+import os
+from collections import OrderedDict
+
+import numpy as np
+
+from mmpose.datasets.builder import DATASETS
+from .face_base_dataset import FaceBaseDataset
+
+
+@DATASETS.register_module()
+class Face300WDataset(FaceBaseDataset):
+    """Face300W dataset for top-down face keypoint localization.
+
+    `300 faces In-the-wild challenge: Database and results.
+    Image and Vision Computing (IMAVIS) 2019`.
+
+    The dataset loads raw images and apply specified transforms
+    to return a dict containing the image tensors and other information.
+
+    The landmark annotations follow the 68 points mark-up.
+
+    Args:
+        ann_file (str): Path to the annotation file.
+        img_prefix (str): Path to a directory where images are held.
+            Default: None.
+        data_cfg (dict): config
+        pipeline (list[dict | callable]): A sequence of data transforms.
+        test_mode (bool): Store True when building test or
+            validation dataset. Default: False.
+    """
+
+    def __init__(self,
+                 ann_file,
+                 img_prefix,
+                 data_cfg,
+                 pipeline,
+                 test_mode=False):
+
+        super().__init__(
+            ann_file, img_prefix, data_cfg, pipeline, test_mode=test_mode)
+
+        self.ann_info['use_different_joint_weights'] = False
+        assert self.ann_info['num_joints'] == 68
+        self.ann_info['joint_weights'] = \
+            np.ones((self.ann_info['num_joints'], 1), dtype=np.float32)
+
+        self.ann_info['flip_pairs'] = [[1, 17], [2, 16], [3, 15], [4, 14],
+                                       [5, 13], [6, 12], [7, 11], [8, 10],
+                                       [18, 27], [19, 26], [20, 25], [21, 24],
+                                       [22, 23], [32, 36], [33, 35], [37, 46],
+                                       [38, 45], [39, 44], [40, 43], [41, 48],
+                                       [42, 47], [49, 55], [50, 54], [51, 53],
+                                       [62, 64], [61, 65], [68, 66], [59, 57],
+                                       [60, 56]]
+
+        self.dataset_name = '300w'
```

Comment:
`300w`, so concise a name

Reply:
300W is the standard abbreviation for this dataset : )

Reply:
okay, `w` means `wild` and not `wan`

Reply:
300 images in the wild   hhhh